src/main/java/seedu/rc4hdb/model/resident/Resident.java

```diff
@@ -154,7 +154,10 @@ public String toString() {
         Set<Tag> tags = getTags();
         if (!tags.isEmpty()) {
             builder.append("; Tags: ");
-            tags.forEach(builder::append);
+            for (Tag t : tags) {
+                builder.append(t + ", ");
+            }
+            builder.delete(builder.length() - 2, builder.length());
```

Comment:
You could probably make use of indices instead to avoid appending the `,` to the last tag.

Reply:
Might just be slightly less readable with this, but we could keep this.